### PR TITLE
Add netlify link to footer

### PR DESCRIPTION
**Pull request by PromoFaux**
Signed-off-by: Adam Warner <me@adamwarner.co.uk>

### CODE REVIEW

File: overrides/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <div class="md-footer-copyright__highlight">
+            {{ config.copyright }}
+          </div>
+        {% endif %}
+        Made with
+        <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
+          Material for MkDocs
+        </a>
+        <br>
+        <a href="https://www.netlify.com">
```
Comment (dschaper) [inline overrides/partials/footer.html:52]:
```suggestion
        <a href="https://www.netlify.com" target="_blank" rel="noreferrer">
```

Comment (dschaper) [reply overrides/partials/footer.html:52]: Open in new tab and don't add the referrer information.

**Comment by XhmikosR** [comment]
Keep in mind that this adds extra maintenance cost. Whenever mkdocs-material is updated one has to check if the template has been updated and manually backport any changes. Just FYI.

**Comment by PromoFaux** [comment]
Will keep that in mind if and when we consume and mkdocs-material updates, thanks.

**Pull request merged**